README.md

```diff
@@ -216,6 +216,12 @@ $ perceval git '/tmp/gitlog.log'
 $ perceval github elastic logstash --from-date '2016-01-01'
 ```
 
+You can pass category as parameter which take `issue`, `pull_request` when using Github Perceval backend to get the specific data.
```

Comment:
Category is a param available for all backends, but in this sentence it seems that that param is valid only for the GitHub backend. I would rephrase the sentence with something like:
```
The GitHub backend accepts the categories `issue`, `pull_request` and `repository` which allow to fetch the specific data.
```

what do you think @vchrombie ?

Reply:
Is the category param available for all the backends?